app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -72,6 +76,17 @@ class RecentPatientsViewController @Inject constructor(
     return Observable.merge(updateRecentPatients, toggleEmptyState)
   }
 
+  private fun addSeeAllIfListTooLong(
+      recentPatients: List<RecentPatientItem>,
+      recentPatientLimit: Int
+  ) =
+      if (recentPatients.size > recentPatientLimit) {
```

Comment:
I just realised that there is a `.take` extension function on `Iterable`, so you can also make this code block a bit more terse:

```kotlin
if (recentPatients.size > recentPatientLimit) {
  recentPatients.take(recentPatientLimit) + SeeAllItem
} else {
  recentPatients
}
```